zhaquirks/thirdreality/motion_sensor.py

```diff
@@ -0,0 +1,37 @@
+"""Third Reality Plug devices."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class ThirdRealityMotionCluster(CustomCluster):
+    """Third Reality's motion sensor private cluster."""
+
+    cluster_id = 0xFF01
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Define the attributes of a private cluster."""
+
+        cooldown_time: Final = ZCLAttributeDef(
+            id=0x0001,
+            type=t.uint16_t,
+            is_manufacturer_specific=True,
+            name="Cooldown_Time",
+        )
+
+
+(
+    QuirkBuilder("Third Reality, Inc", "3RMS16BZ")
+    .replaces(ThirdRealityMotionCluster)
+    .number(
+        attribute_name=ThirdRealityMotionCluster.AttributeDefs.cooldown_time.name,
+        min_value=5,
+        max_value=3600,
+        cluster_id=ThirdRealityMotionCluster.cluster_id,
+    )

```

Comment:
We now need a translation key and fallback name (that'll be the name used by the entity in HA, although it'll be translatable by the translation key – fallback name is just the English default.)
```suggestion
    .number(
        attribute_name=ThirdRealityMotionCluster.AttributeDefs.cooldown_time.name,
        min_value=5,
        max_value=3600,
        cluster_id=ThirdRealityMotionCluster.cluster_id,
        translation_key="cooldown_time",
        fallback_name="Cooldown time",
    )

```